app/controllers/katello/api/v2/systems_controller.rb

```diff
@@ -70,6 +70,12 @@ def index_relation
         collection = System.readable
       end
 
+      if params[:available_for] && params[:available_for] == 'host_collection'
+        system_ids = HostCollection.find(params[:host_collection_id]).systems.pluck(:id)
+        collection = collection.where("id NOT IN (?)", system_ids) unless system_ids.empty?
+        return collection
+      end
```

Comment:
Whats the difference in this block of code and available_for_host_collection function below?